### PR TITLE
add unconference and two unconference planning meetings

### PR DESCRIPTION
**Pull request by bast**
The planning meetings are on the Monday and Tuesday just before the event (Wednesday and Thursday).

### CODE REVIEW

File: calendars/conferences.yaml
Comment (mtomasini) [inline calendars/conferences.yaml:1]: I believe you put 2022 in the Unconference dates! Other than that, it looks good.

File: calendars/conferences.yaml [hunk 1]
```diff
@@ -21,5 +21,18 @@ events:
     begin: 2022-10-19 13:00:00
     duration: { hours: 3 }
 
+  - summary: Unconference 2023
+    description: |
+      Nordic RSE 2023 Unconference. See the event page
+      at https://nordic-rse.org/events/2023-online-unconference/
+      and registration page at https://indico.neic.no/e/nordic-rse-unconference-2023
+    begin: 2022-10-25 13:00:00
```
Comment (bast) [inline calendars/conferences.yaml:29]:
```suggestion
    begin: 2023-10-25 13:00:00
```

File: calendars/conferences.yaml [hunk 2]
```diff
@@ -21,5 +21,18 @@ events:
...
+    duration: { hours: 3 }
 
-
+  - summary: Unconference 2023
+    description: |
+      Nordic RSE 2023 Unconference. See the event page
+      at https://nordic-rse.org/events/2023-online-unconference/
+      and registration page at https://indico.neic.no/e/nordic-rse-unconference-2023
+    begin: 2022-10-26 13:00:00
```
Comment (bast) [inline calendars/conferences.yaml:37]:
```suggestion
    begin: 2023-10-26 13:00:00
```

**Comment by bast** [comment]
Thanks!

**Pull request merged**